class22a/js/script.js

```diff
@@ -0,0 +1,25 @@
+const arr = [10,4,30,13,9,17,2,24,11,23,5,7];
+let arrTemporary=0
+
+//partaA
+function sortArr(arr){
+    for(i=0; i<arr.length; i++) { if (arr[i] > arr[i+1]){ return false;} //Q(n)
+    }
+    return true;
+}
+//PartB
+while(!sortArr(arr)){
+   for(i=0;i<arr.length;i++){if(arr[i]>arr[i+1]){ //Q(n)
```

Comment:
For inside while is make the complexity to be O(n^2), and that's is not solve the task that I ask for